tests/Unit/Numerical/DiscontinuousGalerkin/Actions/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(SPECTRE_UNIT_ACTIONS_TESTS
+    Numerical/DiscontinuousGalerkin/Actions/Test_FluxCommunication.cpp
+   )
+
+set(SPECTRE_UNIT_DISCONTINUOUS_GALERKIN_TESTS "${SPECTRE_UNIT_DISCONTINUOUS_GALERKIN_TESTS};${SPECTRE_UNIT_ACTIONS_TESTS}"
```

Comment:
since this is getting long would you mind breaking the line after `SPECTRE_UNIT_DISCONTINUOUS_GALERKIN_TESTS` ?